pensyve-mcp-gateway/scripts/fetch-model-bundle.sh

```
@@ -0,0 +1,330 @@
+            errors=1
+        else
+            actual_revision="$(cat -- "${root}/${repository}/refs/main")"
+            if [[ "${actual_revision}" != "${revision}" ]]; then
+                echo "floating or mismatched ref rejected: ${repository}/refs/main" >&2
+                errors=1
+            fi
+        fi
+
+        if [[ -d "${root}/${repository}/refs" ]]; then
+            while IFS= read -r path; do
+                path="${path#${root}/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)